packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -75,6 +75,14 @@ class TestCommand extends FlutterCommand {
         help: 'Whether matchesGoldenFile() calls within your test methods should '
               'update the golden files rather than test for an existing match.',
       )
+      ..addFlag('watch',
+        defaultsTo: false,
+        negatable: false,
+        hide: true, // TODO(gamebox): Remove when UX is finalized
+        help: 'After initial run, continue watching application directory for '
```

Comment:
"similar semantics to hot reload" - is that actually true? do constants not change between test runs now?

Reply:
Sorry, I'm not sure what you mean by "do constants not change between test runs now?"  We can discuss offline.  By "similar semantics", I meant we only recompile the changed files and then rerun the test code - not to mention that the rerunning happens by pressing 'r' in the terminal/stdin.  Which is why I did not say "the exact same semantics".

Reply:
Changed the language to reduce confusion.